package.json

```
@@ -14,21 +14,25 @@
 		"dev": " NODE_NO_WARNINGS=1 tsc --watch",
 		"lint": "eslint \"source/**/*.{js,ts,tsx}\"",
 		"lint:fix": "eslint \"source/**/*.{js,ts,tsx}\" --fix",
-		"test": "prettier --check ./source && npx tsx --test test*.tsx",
+		"test": "prettier --check ./source && npx vitest run",
```

The npx is redundant here..

```suggestion
		"test": "prettier --check ./source && vitest run --coverage",
```

---

I removed the coverage subcommand too